setup.cfg

```diff
@@ -72,6 +72,8 @@ exclude =
     mitmproxy/proxy/config.py
     mitmproxy/proxy/server.py
     mitmproxy/proxy/layers/tls.py
+    mitmproxy/proxy/layers/dtls.py
+    mitmproxy/tls.py
```

Comment:
Both the `.coverage.xml` and running `pytest --cov` locally show that this file has 100% coverage but `individual_coverage` says it's at 98%. I'm not sure what the issue is....